src/components/Profile.js

```diff
@@ -142,7 +142,7 @@ Profile.propTypes = {
     patchInfo: PropTypes.func.isRequired
 };
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state) => ({
```

Comment:
why?

Reply:
xD, I added "ownProps", and forgot to remove those.